packages/acceptance-tests/pkg-tests-specs/sources/node-modules.test.ts

```diff
@@ -1827,4 +1831,107 @@ describe(`Node_Modules`, () => {
         ]);
       }),
   );
+
+  testIf(() => process.platform === `win32`,
+    `'nmFolderLinkMode: symlinks' on Windows should use symlinks in node_modules directories`,
+    makeTemporaryEnv(
+      {
+        workspaces: [`ws1`],
+      },
+      {
+        nodeLinker: `node-modules`,
+        nmFolderLinkMode: `symlinks`,
+      },
+      async ({path, run}) => {
+        await writeJson(npath.toPortablePath(`${path}/ws1/package.json`), {
+          name: `ws1`,
+        });
+
+        await run(`install`);
+
+        const {stdout: reparsePoints} = await execPromise(`dir ${npath.fromPortablePath(`${path}/node_modules`)} /al /l | findstr "<SYMLINKD>"`, {shell: `cmd.exe`});
```

Comment:
Couldn't find a cleaner way to distinguish between a `junction` and `symlink` on Windows. Have any ideas?

Reply:
I think there is no cleaner way, as per:
https://github.com/nodejs/help/issues/1520